.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug in Searx
+title: ''
+labels: bug
+assignees: ''
+
+---
+**Version of Searx and commit number if you are using on master branch**
```

Comment:
Looking at searx.space some instances say they are running searx version 0.16.0 and at the same time the google engine is working. It should not be the case, see https://github.com/searx/searx/pull/2236

My guess is there are a lot of custom version.

Reply:
Ok I'll include a notice if one have a fork and should stipulate it.